index.html

```diff
@@ -5,11 +5,147 @@
+                <figcaption class="time"> 3:00 PM</figcaption>
+                <figcaption>We found a teahouse while I was walking down the street. They sold a wide variety of tea. We
+                    could try different kinds of tea for free. </figcaption>
+            </figure>
+        </div>
+        <div class="afterphoto">
+            <figure><img src="images/cafe2.jpg" alt="tea cafe view">
+                <figcaption class="time"> 4:00 PM</figcaption>
+                <figcaption>We found a teahouse while I was walking down the street. They sold a wide variety of tea. We
+                    could try different kinds of tea for free.</figcaption>
+            </figure>
+        </div>
+    </div>
+
+
+    <figure><img class="full" src="images/sunset.jpg" alt="sunset in colorado">
+        <figcaption class="time2"> 5:00 PM</figcaption>
+        <figcaption>Watching sunset in Colorado was one of my most beautiful memory in my life. The vast plain allowed
+            me to enjoy the sunset. </figcaption>
+    </figure>
+
+    <div class="night">
+        <div class="nightphoto">
+            <figure><img src="images/night1.jpg" alt="night view in the car">
+                <figcaption class="time3"> 6:00 PM</figcaption>
+                <figcaption class="time3">Night came to Colorado. My friend drove her car to come back home.
+                </figcaption>
+            </figure>
+        </div>
+        <div class="nightphoto">
+            <figure><img src="images/night2.jpg" alt="night view passing the road ">
+                <figcaption class="time3"> 7:00 PM</figcaption>
+                <figcaption class="time3"> The view I saw during drive feels awkward because the street in baltimore is
+                    busy but street of Colorado was sequestered. </figcaption>
+            </figure>
+        </div>
+        <div class="nightphoto">
+            <figure><img src="images/night3.jpg" alt="calm night">
+                <figcaption class="time3"> 8:00 PM</figcaption>
+                <figcaption class="time3"> When I got out of the car, I felt fresh and cold air. The sky was full of
+                    stars which is hard to see in the city. It was really beautiful.</figcaption>
+            </figure>
+        </div>
+    </div>
+
+    <div class="food">
+        <h2>Food</h2>
+        <figure><img src="images/pizza.jpg" alt="lunch pizza">
+            <figcaption>My friend took me her favorite pizza restaurant "Lucky pie". I ordered
+                prosciutto pizza. It
+                was
+                awesome.
+            </figcaption>
+        </figure>
+        <figure><img src="images/burger.jpg" alt="dinner burger meal">
+            <figcaption>This place is also one of my friend's favorite restaurant, "Lark Burger". I ate famous
+                menu, truffle burger and truffle fries.
+    </div>
+    <figure><img src <script src="https://unpkg.com/aos@next/dist/aos.js">
```

Comment:
There's some broken markup here:
```suggestion
    <script src="https://unpkg.com/aos@next/dist/aos.js">
```

This broken markup is likely why AOS didn't work for you.